packages/lib/hooks/useTheme.tsx

```diff
@@ -33,10 +32,25 @@ export default function useTheme(themeToSet?: Maybe<string>) {
     // because there might be another booking page with conflicting theme.
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [themeToSet, setTheme, embedTheme]);
-  return {
-    resolvedTheme,
-    setTheme,
-    forcedTheme,
-    activeTheme,
-  };
+
+  if (getOnly) {
```

Comment:
As mentioned earlier. We no more return these values unless getOnly is passed as true, so that dev doesn't accidentally use useTheme for getting the values. There is useGetTheme for that.

Also, removed setTheme method because we never want theme to be set from outside as that won't handle embed theme.